cmd/root.go

```diff
@@ -241,6 +322,23 @@ func Execute(cmd *cobra.Command, args []string) {
 		utils.RunIgnoreErr("podman", "rmi", imgid)
 	}
 
+	// Check to see if we need to tune kernel arguments
+	tuningChanged, err := updateTuningArgs(kernelTuningFile)
+	if err != nil {
+		glog.Infof("unable to parse tuning file %s: %s", kernelTuningFile, err)
+	}
+	// If tuning changes but the oscontainer didn't we still denote we changed
+	// for the reboot
+	if tuningChanged {
+		changed = true
+		// Remove the as it's been processed
+		err = os.Remove(kernelTuningFile)
```

Comment:
@cgwalters  We don't _have_ to remove the file. Instead we could verify the last hash of what we checked (or something similar) and NOOP early if no changes are present. Would that alleviate the MCD going degraded issue?

Reply:
Then it becomes a matter of MCD dropping the file per it's MC with "ADD $ARG" or "DELETE $ARG".